docs/en_US/NNICTLDOC.md

```diff
@@ -567,6 +568,54 @@ Debug mode will disable version check function in Trialkeeper.
     nnictl package show
     ```
 
+<a name="feed"></a>
+![](https://placehold.it/15/1589F0/000000?text=+) `nnictl feed`
+
+* Description
+
+  You can use this command to feed several prior or supplementary trial hyperparameters and results for NNI hyperparameter tuning. The data are fed to the tuning algorithm (e.g., tuner or advisor).
+
+* Usage
+
+  ```bash
+  nnictl feed [OPTIONS]
+  ```
+* Options
+
+  |Name, shorthand|Required|Default|Description|
+  |------|------|------|------|
+  |id|  False| |The id of the experiment you want to feed data into|
+  |--file, -f|  True| |a file with data you want to feed in json format|
+
+* Details
+
+  The file with feed data must in json format, an example is shown below:
+
+  ```json
+  [
+    {"paramter": {"x": 0.5, "y": 0.9}, "value": 0.03},
+    {"paramter": {"x": 0.4, "y": 0.8}, "value": 0.05},
+    {"paramter": {"x": 0.3, "y": 0.7}, "value": 0.04}
+  ]
```

Comment:
can I get this json file from nni file?

Reply:
Will figure out a design later or provide a script.

Reply:
Support nni file now.